lib/aws_extensions/elb/PolicyDescription.rb

```diff
@@ -0,0 +1,16 @@
+module AwsExtensions
+  module ELB
+    module PolicyDescription
+      # Public: Convert this Aws::ELB::Types::PolicyDescription
+      # into a json version for migrating to Cumulus
+      def to_cumulus_hash
+        {
+          "type" => self.policy_type_name,
```

Comment:
Yeah I know but I think it's a nicer read so you know the property is coming from the AWS Type and not something local